packages/styles/scss/patterns/blocks/contentgroupcards/_contentgroupcards.scss

```diff
@@ -40,12 +38,10 @@
     }
 
     .#{$prefix}--content-group {
-      @include carbon--breakpoint(sm) {
-        @include carbon--make-col(4, 4);
-        .#{$prefix}--content-group-cards__row {
-          padding-left: $carbon--layout-01;
-          padding-right: $carbon--layout-01;
-        }
+      @include carbon--make-col(4, 4);
+      .#{$prefix}--content-group-cards__row {
+        padding-left: $carbon--layout-01;
+        padding-right: $carbon--layout-01;
```

Comment:
For another pull request we should create an issue to refactor this SCSS so it doesn't retain so much grid in it. It still needs to be able to fit inside of `ContentBlockMixed`